code/modules/mob/mob_grab.dm

```diff
@@ -115,6 +115,8 @@
 		affecting.Weaken(5)	//Should keep you down unless you get help.
 		affecting.losebreath = min(affecting.losebreath + 2, 3)
 
+/obj/item/weapon/grab/attack_self(mob/user)
+	s_click(hud)

```

Comment:
Because without s_click you can't click on the grab to upgrade the grip. Both are required.